workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+---
+name: 'Konami Code'
+description: 'Implement a konami code on a website'
+author: '@faisalsayed10'
+---
+
+# Konami Code
+
+You may have heard about movie Easter Eggs or video game Easter Eggs before, but have you ever heard about website Easter Eggs? The internet is filled with copious amounts of weird and wonderful hidden features! I bet you have played the popular game GTA, right? GTA too had konami codes but they were called 'cheat codes'. 
```

Comment:
```suggestion

You may have heard about the movie Easter Eggs or video game Easter Eggs before, but have you ever heard about the website Easter Eggs? The internet is filled with copious amounts of weird and wonderful hidden features! I bet you have played the popular game GTA, right? GTA too had Konami codes but they were called 'cheat codes'. 
```

Reply:
in this context `the` is not needed, but @faisalsayed10 you might want to make Easter eggs lower case

Reply:
Where have u found this file?
I think this is my very old file, and this isn't what is in the PR

Reply:
@bezlin6mechminerz you have reviewed my very old commit, this is not which is the real PR